po/da.po

```diff
@@ -2631,484 +2633,6 @@ msgid ""
 "columns):"
 msgstr ""
 
-#: src/exercises/day-1/for-loops.md:41
```

Comment:
I'm surprised to see all the messages disappear. Is it because the file is very outdated?

Reply:
None of them were translated, but were causing alignment issues, so I just dropped them.

Although now that I think of it, I searched for `^msgstr ""` which may not have been an appropriate search pattern.  I'll take another look.

Reply:
fixed :)